app/assets/javascripts/services/builds_svc.js

```diff
@@ -0,0 +1,22 @@
+samson.service('buildsService', function($http, $q, httpErrorService) {
+  var config = {
+    headers: {
+      'Accept': 'application/json'
+    }
+  };
```

Comment:
I think an alternative to setting the json header is to append '.json' to the URL. But I don't mind which you use.